TRX_Merger/TestRunMerger.cs

```diff
@@ -63,7 +63,8 @@ private static TestRun MergeTestRuns(List<TestRun> testRuns)
 
             foreach (var tr in testRuns)
             {
-                allResults = allResults.Concat(tr.Results).ToList();
+                if (excludePassedTests) allResults = allResults.Concat(tr.Results.Where(c => c.Outcome != "Passed" || includeExplicitTests.Contains(c.TestName))).ToList();
```

Comment:
sorry i may have slightly misled on this, passed test could be included in the test results, i was looking for an option to exclude output and stdout for passed tests to reduce size, with an option to include stdout for some tests if required for debug purposes 

![image](https://github.com/chanteld-ar/trx-merger/assets/64524819/c7e252b4-3fc3-4d70-af3f-0ba18dcacef4)


Ofcourse ErrorInfo Message and StackTrace is useful for failed tests


this is line 43 of TrxSerializationUtils.cs

Reply:
Done